__template__.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+"""
+${FOGLAMP_LICENSE_DESCRIPTION}
+
+Homepage and documentation: http://foglamp.readthedocs.io/
```

Comment:
I would be tempted to rephrase this as

See: http://fogloamp.readthedocs.io/

This gives us some flexibility if we want to make a separate homepage for the product